Assets/Prefabs/Dodo.prefab

```diff
@@ -96,6 +98,52 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   dodoSpeed: 5
-  cliffBounds: {fileID: 0}
-  mountainBounds: {fileID: 0}
+  sniffRange: 0
   behaviourChangeSpeed: 5
+--- !u!61 &6872853301279111260
+BoxCollider2D:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 6102199197974787306}
+  m_Enabled: 1
+  m_Density: 1
+  m_Material: {fileID: 0}
+  m_IsTrigger: 1
+  m_UsedByEffector: 0
+  m_UsedByComposite: 0
+  m_Offset: {x: 3.5939999, y: 6.7199993}
+  m_SpriteTilingProperty:
+    border: {x: 0, y: 0, z: 0, w: 0}
+    pivot: {x: 0.2, y: 0.1}
+    oldSize: {x: 11.98, y: 16.8}
+    newSize: {x: 1, y: 0.5}
+    adaptiveTilingThreshold: 0.5
+    drawMode: 0
+    adaptiveTiling: 0
+  m_AutoTiling: 0
+  serializedVersion: 2
+  m_Size: {x: 11.98, y: 16.8}
+  m_EdgeRadius: 0
+--- !u!50 &-5505904398498373190
+Rigidbody2D:
+  serializedVersion: 4
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 6102199197974787306}
+  m_BodyType: 1
+  m_Simulated: 1
+  m_UseFullKinematicContacts: 1
```

Comment:
I set it to Kinematic to ignore physics and m_UseFullKinematicContacts in order to trigger on obstacles